<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SrvConfDlg</class>
 <widget class="QDialog" name="SrvConfDlg">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>115</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Server Setting</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>81</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>81</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Ok</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="portEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>41</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ipEdit</tabstop>
  <tabstop>portEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
